src/components/Restaurant.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles'
+import Card from '@material-ui/core/Card/index';
+import CardContent from '@material-ui/core/CardContent/index';
+import Typography from '@material-ui/core/Typography/index';
+import Divider from '@material-ui/core/Divider/index';
+import Rating from 'material-ui-rating';
+import Tooltip from '@material-ui/core/Tooltip/index';
+import PinDrop from '@material-ui/icons/PinDrop.js'
+
+const CUISINE_ICON_CODE = {'american': 65, 'asian': 46, 'bakery': 80,
+                        'fast_food': 87, 'steak': 51, 'sushi': 73, 'vegetarian': 36};
+
+const styles = theme => ({
```

Comment:
should be extracted to a stylesheet

Reply:
actually this is how material-ui documentation uses their 'classes' feature in order to export 'withStyles'
**export default withStyles(styles)(Restaurant);**